### ISSUE TITLE
no-arrow-function-computed-properties' onlyOnThisContexts does not work in apps.

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-cli/eslint-plugin-ember#593]
:(

no idea how to test this in eslint-plugin-ember.

In the app, I have:
```ts
property: computed(() => []);
```
with the config:
```ts
'ember/no-arrow-function-computed-properties': ['error', { onlyOnThisContexts: true }],
```

expected: no lint error

**Comment by NullVoxPopuli** [comment]
nevermind. option is `onlyThisContexts`
